husky_components/dropzone/main.js

```diff
@@ -284,22 +284,26 @@ define([], function() {
                         this.addFiles(event.originalEvent.dataTransfer.files);
                     }
                 }.bind(this));
+                this.$dropzone.on('click', function() {
+                    if (!!this.options.cancelUploadOnOverlayClose) {
```

Comment:
That's what the configuration is about. The upload's should only be canceled if the `cancelUploadOnOverlayClose` is set and the overlay is closed with a click by the user. If this piece of code was in the `closeOverlay` method, the uploaded files would also get canceled when they have been successfully uploaded (because the overlay closes after that). From the server perspective everything is fine, however the dropzone in the frontend views the uploaded files as canceled and triggers, that no files have been uploaded.